wingtips-servlet-api/README.md

```diff
@@ -72,13 +87,16 @@ See the [base project readme's section on propagation](../README.md#propagating_
 propagating tracing information. You may also want to consider
 [wrapping downstream calls in a subspan](../README.md#sub_spans_for_downstream_calls).
 
-## NOTE - Servlet API 3.0.1 or later dependency required at runtime
+## NOTE - Servlet API dependency required at runtime
 
-This `wingtips-servlet-api` module has a minimum Servlet 3.0.1 requirement, but does not export any transitive Servlet 
-API dependencies to prevent runtime version conflicts with whatever Servlet environment you deploy to. 
+This `wingtips-servlet-api` module does not export any transitive Servlet API dependencies to prevent runtime version 
+conflicts with whatever Servlet environment you deploy to. 
 
 This should not affect most users since this library is likely to be used in a Servlet environment where the Servlet 
-API is on the classpath at runtime, however if you receive `NoClassDefFoundError`s related to Servlet API classes then 
-you'll need to pull a Servlet API dependency into your project that supports a minimum Servlet 3.0.1 version. For 
-reference, `wingtips-servlet-api` uses the compile-only Servlet API dependency 
-[`javax.servlet:javax.servlet-api:3.0.1`](http://search.maven.org/#artifactdetails%7Cjavax.servlet%7Cjavax.servlet-api%7C3.0.1%7Cjar). 
+API is on the classpath at runtime, however if you receive class-not-found errors related to Servlet API classes then 
+you'll need to pull a Servlet API dependency into your project. Library authors who wish to build on functionality in
+this module might need to do this. Which Servlet API dependency you pull in depends on the type of Servlet environment 
+you want to support (Servlet 2.x or Servlet 3+). For example:
+
+* Servlet 3 API dependency: [`javax.servlet:javax.servlet-api:$servlet3ApiVersion`](http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22javax.servlet%22%20AND%20a%3A%22javax.servlet-api%22) 
```

Comment:
Variables intentional?

Reply:
Yes, but I can see how it looks like a copy/paste error. I'll see if I can clear it up.